addons/mail/static/src/views/form/form_renderer.scss

```diff
@@ -25,11 +25,6 @@
         }
     }
 
-    &.o-isInFormSheetBg {
```

Comment:
Thanks @Brieuc-brd , this solves the bg-color issue indeed, but exposes another one since chatter and form's content are not aligned anymore.

![image](https://github.com/odoo/odoo/assets/8558351/f3c6a06e-d6e9-48ef-9e34-c72471acd7cf)



I propose something like:
```scss
&.o-isInFormSheetBg {
    margin-left: calc(var(--formView-sheetBg-padding-x) * -1);
    margin-right: calc(var(--formView-sheetBg-padding-x) * -1);
}
```

Reply:
@stefanorigano Changed made 👍 I also added a `w-auto` on the container to make it work.